# This Python script will fail deliberately

 fail():
    raise xception("This is a deliberate failure!")

fail()
